src/test/java/org/prebid/server/bidder/pubmatic/PubmaticBidderTest.java

```diff
@@ -644,6 +626,71 @@ public void makeBidsShouldReturnBannerBidIfExtBidContainsIllegalBidType() throws
                 .containsOnly(BidderBid.of(Bid.builder().impid("123").ext(bidType).build(), banner, "USD"));
     }
 
+    @Test
+    public void makeHttpRequestsShouldReplaceDctrIfPresent() throws IOException {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                identity(),
+                extImpPubmaticBuilder -> extImpPubmaticBuilder
+                        .dctr("dctr")
+                        .keywords(singletonList(
+                                ExtImpPubmaticKeyVal.of("key_val", asList("value1", "value2")))));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = pubmaticBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getPayload)
+                .flatExtracting(BidRequest::getImp)
+                .extracting(Imp::getExt)
+                .containsOnly(mapper.readValue("{\"key_val\":\"dctr\"}", ObjectNode.class));
```

Comment:
Let's first create expected map and than convert it, also instead of containsOnly use ContainsExactly.
So result will look like 
```java
 final Map<String, String> expectedKeyWords = singletonMap("key_val", "dctr");
        assertThat(result.getErrors()).isEmpty();
        assertThat(result.getValue()).hasSize(1)
                .extracting(HttpRequest::getPayload)
                .flatExtracting(BidRequest::getImp)
                .extracting(Imp::getExt)
                .containsExactly(mapper.convertValue(expectedKeyWords, ObjectNode.class));
```
Pls prepare this changes for every test